src/scripts/ui/inline-roll-links.ts

```diff
@@ -315,7 +315,7 @@ export const InlineRollLinks = {
 
                 const actor = resolveActor(foundryDoc, link);
                 if (actor || pf2Traits) {
-                    const origin: Record<string, unknown> = {};
+                    const origin = (flags.pf2e.origin as Record<string, unknown>) ?? {};
```

Comment:
Don't assert

Reply:
https://en.wikipedia.org/wiki/Assertion_(software_development)

Reply:
![image](https://github.com/foundryvtt/pf2e/assets/705879/d05ae3c2-c621-40b1-8d98-f3ef5bdb8b82)

Reply:
![image](https://github.com/foundryvtt/pf2e/assets/705879/1cab6292-561f-45e4-9ac0-b5d60879ebcb)

Reply:
![image](https://github.com/foundryvtt/pf2e/assets/705879/87f9e6d9-0e26-4a5c-bc85-dbc761b85350)

Reply:
I'm trying to find a way to rewrite this whole thing so explicit casting isn't required